mira/metamodel/templates.py

```diff
@@ -18,6 +18,17 @@
 SCHEMA_PATH = HERE.joinpath("schema.json")
 
 
+class Config(BaseModel):
+    prefix_priority: List[str]
+
+
+DEFAULT_CONFIG = Config(
+    prefix_priority=[
+        "ido",
```

Comment:
@bgyori this is where we can keep our prefix priority list, but ultimately this should be configurable